web/src/popover_menus.js

```diff
@@ -82,6 +82,13 @@ export function is_time_selected_for_schedule() {
 }
 
 export function get_selected_send_later_time() {
+    if (!selected_send_later_time) {
+        return undefined;
+    }
+    return selected_send_later_time;
+}
+
+export function get_formatted_selected_send_later_time() {
```

Comment:
Are we concerned with the "at" in this string for non-English situations?

Reply:
Yeah, changed this to use this:
```
-    return format(new Date(selected_send_later_time), "MMM d yyyy 'at' h:mm a");
+    return get_full_datetime(selected_send_later_time);
```

@amanagr in general we should always be using the `timerender` helpers rather than `date-fns` directly at this point.

Reply:
Hmm, that probably includes seconds; did a few more tweaks here.